package.json

```diff
@@ -12,9 +12,12 @@
     "babel-preset-es2015": "^6.24.0",
     "babel-register": "^6.24.0",
     "coffee-script": "~1.8.0",
+    "cookiejar": "^2.1.1",
+    "csprng": "^0.1.2",
     "dotenv": "^4.0.0",
     "gists": "^0.1.2",
     "github-api": "^3.0.0",
+    "github-base": "^0.5.4",
```

Comment:
same here? why do we need `github-base`

Reply:
@zelima It was installed after lint result. It is used in `messages-test.js`

Reply:
@Mikanebu I really don't get this? XO installed this package?

Reply:
@zelima I installed, it errored about installing this package

Reply:
@Mikanebu after doing what? Is it related to XO, or you just forgot to install in previous commits? if so why it was not erroring on travis?

Reply:
@zelima after running `lint`, it asked to install it with providing `npm install --S github-base`.

Reply:
So as I understood, XO just recommended installing `github-base` but we do not know the reason. That's strange. What if we do not do, will tests fail?

Reply:
@zelima now it errors for `Cannot find module 'github-base'`